contentcuration/contentcuration/frontend/shared/views/details/DetailsRow.vue

```diff
@@ -0,0 +1,53 @@
+<template>
```

Comment:
Moved from frontend/preview to better match how we're handling shared components